src/Razor/src/Microsoft.AspNetCore.Razor.Language/Legacy/ParserHelpers.cs

```diff
@@ -121,5 +121,3752 @@ public static bool IsDecimalDigit(char value)
             "track",
             "wbr"
         };
+
+        // From https://www.w3.org/TR/html5/syntax.html#named-character-references
+        public static readonly Dictionary<string, string> HtmlEntities = new Dictionary<string, string>()
+        {
+            { "&Aacute;", "\u00C1" },
```

Comment:
Did you generate this or type it all in? It seems like it would be somewhat valuable to be able to generate this.

Reply:
Definitely did not type in 3500 things 😄. I wrote a console app to parse this JSON https://www.w3.org/TR/html5/entities.json and spit it out in this format.